modules/distributor/distributor.go

```diff
@@ -376,6 +376,9 @@ func (d *Distributor) sendToIngestersViaBytes(ctx context.Context, userID string
 		op = ring.Write
 	}
 
+	responses := make([]*tempopb.PushResponse, 0, d.ingestersRing.ReplicationFactor())
```

Comment:
won't this be the count of all ingesters? one response for each ingester? not sure how easy it is to get that number.